server/src/cloud/column.ts

```diff
@@ -18,77 +20,85 @@ export interface DeleteColumnRequest {
 
 export interface EditColumnRequest {
   boardId: string;
-  columnId: string;
-  name?: string;
-  color?: Color;
-  hidden?: boolean;
+  column: {
+    columnId: string;
+    name?: string;
+    color?: Color;
+    hidden?: boolean;
+  };
 }
 
 export const initializeColumnFunctions = () => {
-  api<AddColumnRequest, boolean>("addColumn", async (user, request) => {
+  api<AddColumnRequest, {status: string; description: string}>("addColumn", async (user, request) => {
```

Comment:
I think you should put the `{status: string; description: string}` in a general response type since it's used on multiple occasions.

Reply:
Good idea. Replaced with `StatusResponse` 👍